edgexfoundry/app-service-configurable
# feat: Enable use of common app service metrics

closes #399

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  **TBD**

## Testing Instructions
Build and run this branch with DEBUG logging and with edgex non-secure stack running, stop `app-rules-engine`
Run with each profile.
Verify those with 0s Interval log message about reporting disabled
Verify those with 30s Interval log message about reporting 0 metrics every 30 secs
In a the sample profile enable all 3 metrics
Verify log message about reporting 7 metrics every 30 secs (this is 2 for the 3 pipelines each and the genal metric)

## New Dependency Instructions (If applicable)
